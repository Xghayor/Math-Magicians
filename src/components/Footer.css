
.footer {
  background-color: #333;
  color: white;
  text-align: center;
  padding: 0.1rem;
}
