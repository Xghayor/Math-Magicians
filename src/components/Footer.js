import React from 'react';
import './Footer.css'; // Import your custom styles

const Footer = () => (
  <footer className="footer">
    <p>&copy; Microverse 2023</p>
  </footer>
);

export default Footer;
