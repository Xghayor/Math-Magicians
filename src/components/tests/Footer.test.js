import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../Footer';

test('Footer component matches snapshot', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
