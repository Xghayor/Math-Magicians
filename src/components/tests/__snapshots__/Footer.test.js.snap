// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <p>
      Â© Microverse 2023
    </p>
  </footer>
</DocumentFragment>
`;
